docs/content/library/AsyncArrow.fsx

```diff
@@ -0,0 +1,339 @@
+(**
+
+# F# Async: AsyncArrow
```

Comment:
I think I much prefer the name ``AsyncFunc``.  Is there any reason not to use that?  I think of it as almost standard F# terminology - "an asynchronous function is a function ``T -> Async<U>``"